sql/testdata/builtin_function

```diff
@@ -477,6 +477,44 @@ SELECT radians(-45.0), radians(45.0)
 ----
 -0.7853981633974483 0.7853981633974483
 
+# Rounding to negative number of digits is not Postgres compatible.
```

Comment:
"Our implementation of round is not fully Postgres-compatible because we do not allow negative numbers of digits."